health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppModule.js

```
@@ -29,23 +138,59 @@
+        actionFields={[
+          <Button
+            label={t("GO_BACK")}
+            title={t("GO_BACK")}
+            variation="secondary"
+            style={{
+              marginLeft: "2.5rem"
+            }}
+            onClick={() => {
+              history.push(`/${window.contextPath}/employee/campaign/view-details?campaignNumber=${campaignNumber}`);
+            }}
+          />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)